plugins/azure-functions-backend/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@backstage/plugin-azure-functions-backend",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.0.0",
```
When we release the changeset will make it 0.1.0